printdb.py

```diff
@@ -0,0 +1,68 @@
+import os
+import sqlite3
+
+from DAO import _Employees, _Suppliers, _Products, _CoffeeStands, _Activities
+from DTO import *
+
+
+def print_employees_report(cursor):
+    print("Employees report")
+    cursor.execute("""SELECT emp.name, emp.salary, cs.location, sum(prod.price * act.quantity)
+    FROM ((Employees AS emp
+    INNER JOIN Coffee_stands AS cs 
+    ON emp.coffee_stand = cs.id)
+    LEFT OUTER JOIN (Activities AS act
+    INNER JOIN Products AS prod ON act.product_id = prod.id)
```

Comment:
Shouldn't the left outer join be replaced with inner join to remove cases of employees with no activities?

Reply:
no. we want to print them as well. notice the new commit with the ifnull()
to make sure it says 0 and not None